lib/modules/job/widgets/jobCard.widget.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class JobCardWidget extends StatefulWidget{
+  @override
+  State<StatefulWidget> createState() {
+    return JobCard();
+  }
+
+}
+
```

Comment:
Vad ca ai facut widget-urile stateful si ecranele stateless.
Stateless = Widget care nu isi manage-uieste state-ul (toate field-urile lui sunt finale)
Stateful = Widget care isi manage-uieste state-ul printr-un State class (in state class poti avea field-uri non final si se le modifici cu `setState(() {...})`)

Cel mai simplu e sa pastrezi un widget stateless pana cand ajungi in punctul in care ai nevoie de un field non-final in el.

Ex.
JobMainScreen - Stateful
- nu are field-uri, dar in general widget-urile care se ocupa de apeluri API sunt stateful (poveste mai lunga)
- cheama fetchJobs() din JobsProvider in initState() {...}
- initializeaza un JobMain cu lista de jobs luata din JobsProvider

JobMain - Stateless
- final List<Job> (primit in constructor de la JobMainScreen)

JobCard - Stateless
- final Job (primit in constructor de la JobMain)